### PR TITLE
WIP - Add station API

### PR DESCRIPTION
**Pull request by PlugaruT**

### CODE REVIEW

File: constants.py [hunk 1]
```diff
@@ -6,4 +6,4 @@
 QOS_MAYBE = 0
 
 
-FORMAT_TIME = '%Y-%m-%dT%H:%M:%SZ'
+FORMAT_TIME = "%Y-%m-%dT%H:%M:%SZ"
```
Comment (ston1x) [inline constants.py:9]: What was the purpose of changing this? Just wondering 🤔

**Comment by ston1x** [comment]
Related to #2

File: constants.py (hunk 1 above)
Comment (PlugaruT) [reply constants.py:9]: `black` did, not me

**Comment by roata** [comment]
@PlugaruT got any news about this? I'm about to add "route-specific last known locations" and I don't know whether I should do it on top of your changes or the current master.

**Comment by PlugaruT** [comment]
I totally forgot about this PR 😬 Soo, I think you can just branch of from master unless there is something useful here. Maybe I'll finish this some day, but right now, I don't really have energy to do this now 😞

**Comment by roata** [comment]
Ok, the updates were made on top of master, but they follow your changes (except moving the subscriber to another file) and `black` was applied to auto-format it.

If you're still looking for a job, consider the possibility of mentioning your open source contributions, which can speak for themselves. A history of commits might offer an advantage over other candidates, because your code gives you a "foot in the door" factor, especially if the employer is a supporter of FLOSS.

For example, I've configured several automated processes with "Github Actions" so now I can legitimately claim that I've applied the tool. Well, you got the idea!

Another thing that might help is to harvest some karma on your Stackoverflow account, as some employers look for that too.

**Pull request closed without merging**